pages/common/ack.md

```diff
@@ -6,9 +6,9 @@
 
 `ack {{foo}}`
 
-- Find files in a specific language:
+- Find files scoped to a specific programming language:
```

Comment:
Pardon my ignorance, but what does "scoped" exactly mean here ? Will it only check files with ".rb" extension ? Or does it mean something else ?

Reply:
Pretty much. From the docs: 

> The following is the list of filetypes supported by ack.  You can specify a file type with the --type=TYPE format, or the --TYPE format.  For example, both --type=perl and --perl work.

> Note that some extensions may appear in multiple types.  For example, .pod files are both Perl and Parrot.

> --[no]ruby         .rb .rhtml .rjs .rxml .erb .rake .spec; Rakefile; first line matches /^#!.*\bruby/

Reply:
Then I think the previous description was clearer. By "scope", I was thinking of "variable scoping" and was wondering how does this apply to files.

Reply:
Made the changes myself.